arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'A gentle introduction: BOLD'
+sidebar_label: 'A gentle introduction'
+description: 'An educational introduction that provides a high-level understanding of BOLD, a new dispute protocol to enable permissionless validation for Arbitrum chains.'
+user_story: 'As a user or researcher of the Arbitrum product suite, I want to learn about BOLD, a next-generation dispute protocol that enables permissionless validation.'
+content_type: gentle-introduction
+author: leeederek
+sme: leeederek
+---
+
+import PublicPreviewBannerPartial from './partials/_bold-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+This introduction is for those who want to learn about BOLD: a new dispute protocol for Optimistic Rollups that can enable **permissionless validation for Arbitrum chains**. BOLD stands for Bounded Liquidity Delay and is currently deployed on a public testnet for anyone to join and test how challenges will work. 
+
+This next-generation dispute protocol technology will soon be available for any Arbitrum chain, and pending a governance vote, will eventually be made available on Arbitrum Sepolia, Arbitrum One, and Arbitrum Nova.
+
+BOLD will eventually replace the current, permissioned fraud proof mechanism that powers Arbitrum chains today.
+
+## In a nutshell:
+
+- BOLD, an acronym for Bounded Liquidity Delay, is a new challenge protocol for Arbitrum chains that ensures disputes will take, at most, 1 challenge period (6.4 days) to get resolved. BOLD effectively eliminates delay attacks by enforcing a upper time-bound for dispute resolution.
+- Validation for Arbitrum One and Nova is currently is limited to a [permissioned set of parties maintained by the Arbitrum DAO](https://docs.arbitrum.foundation/state-of-progressive-decentralization#allowlisted-validators) to reduce the risks of *[delay attacks](https://medium.com/offchainlabs/solutions-to-delay-attacks-on-rollups-434f9d05a07a)* where malicious entities can indefinitely open disputes and delay confirmations of assertions about Arbitrum states.
```

Comment:
Should we also link to the description of the rollup protocol for some extra context?
https://arbitrum-docs-git-derek-bold-gentle-intro-docs-offchain-labs.vercel.app/inside-arbitrum-nitro/#arbitrum-rollup-protocol